### PR TITLE
导入 VMess => 导入分享链接

### PR DESCRIPTION
**Pull request by DuckSoft**
generalise vmess links => share links.
updated some descriptions.

### CODE REVIEW

**Comment by DuckSoft** [comment]
Ready for review. Feel free to correct by committing directly.
Squash before merging.

File: docs/en/getting-started/step3.md [hunk 1]
```diff
@@ -5,21 +5,24 @@ title: Importing Hosts into Qv2ray
 # Step 3: Importing Hosts into Qv2ray
 Qv2ray supports many ways of importing your configuration. To get you started, we have just picked these most common use cases.
 
-## VMess Link
-> VMess links generally looks like this: `vmess://eyJ2IjoiMiIsxxxxqcC5x`...
+## Share Links
+
+> Share links generally looks like these: 
+> * `vmess://eyJ2Ijoi`...
```
Comment (U-v-U) [inline docs/en/getting-started/step3.md:11]: ~~VMess share links do not look like this~~

Comment (U-v-U) [inline docs/en/getting-started/step3.md:11]: VMess share links do not look like this

**Review by U-v-U — CHANGES_REQUESTED** [review]

File: docs/en/getting-started/step3.md (hunk 1 above)
Comment (DuckSoft) [reply docs/en/getting-started/step3.md:11]: You are free to make changes.

**Review by EpLiar — COMMENTED** [review]
Do VMess links need to use the new version?

**Pull request merged**